playbooks/robusta_playbooks/common_actions.py

```diff
@@ -0,0 +1,28 @@
+from robusta.api import *
+
+
+class FindingOverrides(ActionParams):
+    title: str = ""
+    description: str = ""
+    severity: str = ""
+
+    """
+    :var title: Overriding finding title.
+    :var description: Overriding finding description.
+    :var severity: Overriding finding severity. Allowed values: DEBUG, INFO, LOW, MEDIUM, HIGH
+    """
+
+
+@action
+def customise_finding(event: ExecutionBaseEvent, params: FindingOverrides):
+    """
+    Overrides the created finding attribute with the provided attributes overrides.
```

Comment:
I think this is easier to read as:

Overrides a finding attribute with the provided value

Reply:
Perhaps add a sentence explaining what Findings are. E.g. `All messages from Robusta are represented as a Finding object. This action lets you override Finding fields to change that messages Robusta sends. This lets you modify messages created by other actions without needing to rewrite those actions.`

Reply:
done